pkgs/applications/editors/vscode/vscode.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, lib, callPackage, fetchurl, fetchpatch, ... } @ args:
+
+let
+  inherit (stdenv.hostPlatform) system;
+
+  plat = {
+    "i686-linux" = "linux-ia32";
+    "x86_64-linux" = "linux-x64";
+    "x86_64-darwin" = "darwin";
+  }.${system};
+
+  archive_fmt = if system == "x86_64-darwin" then "zip" else "tar.gz";
+
+  sha256 = {
+    "i686-linux" = "0n2k134yx0zirddi5xig4zihn73s8xiga11pwk90f01wp1i0hygg";
+    "x86_64-linux" = "0ljijcqfyrfck5imldis3hx9d9iacnspgnm58kvlziam8y0imwzv";
+    "x86_64-darwin" = "00fg106rggsbng90k1jjp1c6nmnwni5s0fgmbz6k45shfa3iqamc";
+  }.${system};
+in
+  callPackage ./generic.nix (args // rec {
+
+    version = "1.33.1";
+    pname = "vscode";
+
+    # Same here, I have trouble getting isInsiders, and I don't understand its purpose since it is not used anywhere else in the repo
```

Comment:
Need to keep `isInsiders`. I think it is being used as `vscode.override { isInsiders = true; }`.